modules/insights-operator-register-unconnected-cluster.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc
+// * sd_support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc
+
+
+:_content-type: PROCEDURE
+[id="insights-operator-register-unconnected-cluster_{context}"]
+= Registering your unconnected cluster
+
+Register your unconnected {product-title} cluster on the {hybrid-console} so that your cluster is not impacted by the consequences listed in the section named "Consequences of disabling remote health reporting".
+
+By registering your unconnected cluster, the {product-title} Telemetry service can provide metrics about your cluster's health and update status.
```

Comment:
IDK that this statement is accurate (if your unconnected) Telemtery can't report back to us (unless you provide it a way (with a proxy and some connection) to connect back to us.

Reply:
It might be better to say by registering your cluster your ensuring that proper entitlement usage reporting happens; and your not repeatedly asked by your account team for an audit... (while we can't say it this way; this is done for proper reporting and adherence to our subscription terms).

Reply:
Updated with the following notice:

> By registering your unconnected cluster, you can continue to report your subscription usage to Red Hat. In turn, Red Hat can return accurate usage and capacity trends associated with your subscription, so that you can use the returned information to better organize subscription allocations across all of your resources.